### PR TITLE
Path correction for #6351

### PR DESCRIPTION
**Pull request by yophilav**
- Use git `--force` 
- Use relative path to source helper bash functions.

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [x] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [x] Title of the pull request is clear and informative.
- [x] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [x] concise summary of tests added/modified
	- [x] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

File: scripts/linux/publishReleasePackages.sh [hunk 1]
```diff
@@ -253,8 +253,7 @@ publish_to_github()
     branch_name=${BRANCH_NAME/"refs/heads/"/""}
     echo "Branch Name is $branch_name"
 
-    # Note: $(pwd) is the $(Build.SourcesDirectory)
-    source ./iotedge/scripts/linux/github/updateLatestVersion.sh
+    source "$(dirname "$(realpath "$0")")/github/updateLatestVersion.sh"
```
Comment (and-rewsmith) [inline scripts/linux/publishReleasePackages.sh:256]: What does this $0?

File: scripts/linux/publishReleasePackages.sh [hunk 2]
```diff
@@ -253,8 +253,7 @@ publish_to_github()
     branch_name=${BRANCH_NAME/"refs/heads/"/""}
     echo "Branch Name is $branch_name"
```
Comment (and-rewsmith) [inline scripts/linux/publishReleasePackages.sh:254]: Above on line 230:
> If SKIP_UPLOAD=false, the script creates a DRAFT github release page on /azure-iotedge

Should it be the other way around?

File: scripts/linux/github/updateLatestVersion.sh [hunk 3]
```diff
@@ -257,6 +257,6 @@ github_update_and_push()
     lastCommitHash=$(git log -n 1 --pretty=format:"%H")
     git tag "$VERSION" $lastCommitHash
 
-    git push https://$GITHUB_PAT@github.com/Azure/azure-iotedge.git
-    git push https://$GITHUB_PAT@github.com/Azure/azure-iotedge.git "$VERSION"
+    git push --force https://$GITHUB_PAT@github.com/Azure/azure-iotedge.git
+    git push --force https://$GITHUB_PAT@github.com/Azure/azure-iotedge.git "$VERSION"
```
Comment (and-rewsmith) [inline scripts/linux/github/updateLatestVersion.sh:261]: Why force?

**Review by and-rewsmith — APPROVED** [review]

File: scripts/linux/publishReleasePackages.sh (hunk 1 above)
Comment (yophilav) [reply scripts/linux/publishReleasePackages.sh:256]: $0 is the bash input argument for the current running program.
Basically, get the path of the running program, get the directory, and append it with `/github/*` to formulate a relative path

File: scripts/linux/publishReleasePackages.sh (hunk 2 above)
Comment (yophilav) [reply scripts/linux/publishReleasePackages.sh:254]: Good catch! thanks!

File: scripts/linux/github/updateLatestVersion.sh (hunk 3 above)
Comment (yophilav) [reply scripts/linux/github/updateLatestVersion.sh:261]: Because git hub will ask "are you sure?" if you don't force it. It will end up not pushing it.

**Review by and-rewsmith — APPROVED** [review]

**Pull request merged**